### ISSUE TITLE
[Feature Request] add preferenced (weighted?) votes (optional)

### ISSUE THREAD

**Issue by natrius** [nextcloud/polls#670]
**Is your feature request related to a problem? Please describe.**

I was creating a poll where it would be nice to weight the votes.

**Describe the solution you'd like**

 So, A get 4 points, B gets 3, C gets 1 point and D gets 2 points. So in the end the vote with the most points will win. Basically, get points as much as the available choices. A-G you get 7 points for example. 

**Describe alternatives you've considered**

It may be niche, so i'm not completely sure its worth implementing something like this. But it would be unique and could be of value sometimes :)

**Comment by dartcafe** [comment]
This wold mean, we need different Vote types: 
- Yes, No, Maybe (current and only vote type)
- Limited Numer of votes per participant #162 
- Limited Numer of votes per Option #403 
- Priority polls from 1 .. n (n = Numer of Options)
- ... 

will put it in the backlog and think about it...

**Comment by natrius** [comment]
Yes, you are correct. #162 is a voting system, his "yes/no/maybe" choices confused me a bit. So its a bit like "Do you want to use Nextcloud Polls or Google Doodle? Polls | Doodle" and done. #403 is also mentioned in #162 basically, i have such an option. I have to organize an event where a max of 6 people would be possible at a time (Exit-The-Room) But i would like to send it to multiple people out. Right now not really possible.
Any mine would be the priority-polls with 1...n. 

That sums it up pretty good. Ouf, seems like a lot of work. Thank you for what you are doing!

**Comment by dartcafe** [comment]
I think, this is something, what is discussed here: https://github.com/nextcloud/polls/issues/513#issuecomment-461158336
#513

**Comment by dartcafe** [comment]
@natrius see above...

**Comment by natrius** [comment]
Well, its quite similar. In his he can have 7 questions and you have every answer-option on every question. I suggest something like 7 Questions and the first vote you give is given automatically the 1. The second vote you give is automatically 2. Or at least the 1 is not available to choose anymore for voting. (Or maybe the other way around, start with 7 because what gets the most points is more wanted by the majority. Probably makes more sense)
